server/manager/src/main/java/org/apache/accumulo/manager/Manager.java

```diff
@@ -580,7 +559,7 @@ public void run() {
     }
   }
 
-  private boolean shouldCleanupMigration(TabletMetadata tabletMetadata) {
+  boolean shouldCleanupMigration(TabletMetadata tabletMetadata) {
```

Comment:
Could potentially move the migration cleanup thread into BalanceManager (and move this method as well)